server/datastore/mysql/activities.go

```diff
@@ -21,7 +21,7 @@ func (ds *Datastore) NewActivity(ctx context.Context, user *fleet.User, activity
 		user.ID,
 		user.Name,
 		activityType,
-		detailsBytes,
+		string(detailsBytes),
```

Comment:
Just for context, why did we need to cast to string when setting the interpolateParams parameter?

Reply:
There was an error with the type being binary not being able to use it properly with the format string.